recipe/build-mlflow.bat

```diff
@@ -26,7 +13,15 @@ if [%PKG_NAME%] == [mlflow-ui] (
   popd
 )
 
-%PREFIX%/python.exe -m pip install . --no-deps --ignore-installed -vv
+if [%PKG_NAME%] == [mlflow-skinny] (
+  set MLFLOW_SKINNY=1
+  # https://github.com/mlflow/mlflow/pull/4134
+  copy %RECIPE_DIR%/README_SKINNY.rst %SRC_DIR%
```

Comment:
```suggestion
```

Can we remove this? `README_SKINNY.rst` doesn't exist anymore.

Reply:
Sure, but I wonder why this line does not let the pipeline fail if README_SKINNY.rst does not exist.

Reply:
Just by looking at the pipeline log, the file seems to exist
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1199774&view=logs&j=d818e662-7181-5ce9-b265-50d03742b7af&t=dc9b07c2-72db-585f-80fc-2b606081ad2b&l=984